spread.yaml

```diff
@@ -61,7 +61,7 @@ backends:
                 kernel: GRUB 2
                 workers: 4
             - fedora-25-64:
-                workers: 1
+                workers: 2
```

Comment:
It is running 130 tasks, just 1 worker will delay the whole execution, see the log: https://paste.ubuntu.com/25113080/